### ISSUE TITLE
Quickstart with Kind is not working

### ISSUE THREAD

**Issue by tobernguyen** [flanksource/karina#651]
I follow the Kind Quickstart (https://karina.docs.flanksource.com/admin-guide/provisioning/kind/). In step 5 (deploy the bare minimum configuration), I got some error log from karina cli

```
...
ERRO[0046] Failed to deploy base: cert-manager: failed to deploy cert-manager: failed to get dynamic client for &{map[apiVersion:cert-manager.io/v1 kind:ClusterIssuer metadata:map[name:default-issuer namespace:cert-manager] spec:map[ca:map[secretName:cert-manager-webhook-ca]]]}: no matches for kind "ClusterIssuer" in version "cert-manager.io/v1"
...
```

After the provision command, I checked pods in Kind cluster and noticed that `calico-node` is failing to start. Here is log from calico-pod:
```
2021-02-06 02:40:45.836 [INFO][10] startup.go 256: Early log level set to info
2021-02-06 02:40:45.839 [INFO][10] startup.go 272: Using NODENAME environment for node name
2021-02-06 02:40:45.839 [INFO][10] startup.go 284: Determined node name: test-cluster-control-plane
2021-02-06 02:40:45.840 [INFO][10] k8s.go 228: Using Calico IPAM
2021-02-06 02:40:45.840 [INFO][10] startup.go 316: Checking datastore connection
2021-02-06 02:40:45.848 [INFO][10] startup.go 340: Datastore connection verified
2021-02-06 02:40:45.849 [INFO][10] startup.go 95: Datastore is ready
2021-02-06 02:40:45.851 [INFO][10] customresource.go 100: Error getting resource Key=GlobalFelixConfig(name=CalicoVersion) Name="calicoversion" Resource="GlobalFelixConfigs" error=the server could not find the requested resource (get GlobalFelixConfigs.crd.projectcalico.org calicoversion)
2021-02-06 02:40:45.856 [INFO][10] startup.go 584: Using autodetected IPv4 address on interface eth0: 172.17.0.2/16
2021-02-06 02:40:45.856 [INFO][10] startup.go 647: No AS number configured on node resource, using global value
2021-02-06 02:40:45.857 [INFO][10] startup.go 149: Setting NetworkUnavailable to False
2021-02-06 02:40:45.877 [INFO][10] startup.go 536: CALICO_IPV4POOL_NAT_OUTGOING is true (defaulted) through environment variable
2021-02-06 02:40:45.877 [INFO][10] startup.go 796: Ensure default IPv4 pool is created. IPIP mode: Never, VXLAN mode: Never
2021-02-06 02:40:45.879 [ERROR][10] startup.go 802: Failed to create default IPv4 IP pool: 192.168.0.0/16 error=resource does not exist: IPPool(default-ipv4-ippool) with error: the server could not find the requested resource (post IPPools.crd.projectcalico.org)
2021-02-06 02:40:45.880 [WARNING][10] startup.go 1057: Terminating
Calico node failed to start
stream closed
```

All other pods stuck in pending state as calico couldn't setup the CNI.

But if I ignore those errors and continue to do step 6 (`karina deploy all -c test-cluster.yml`) then things start to come up healthy. After step 6 is finished, there are still 2 workloads are not happy which are: dex and platform-operator with `ImagePullBackOff` error.

The image for dex is `quay.io/dexidp/dex:v2.27.0` and for platform-operator is `docker.io/flanksource/platform-operator:0.5.1`.

**Is any of these errors expected? And what is the expected outcome for this Kind Quickstart?**

---

Karina version: `-20210202164958`
Kind version: `kind v0.9.0 go1.15.2 darwin/amd64`

**Comment by BrendanGalloway** [comment]
@tobernguyen The initial error is because there is a new `crds` phase that also needs to be applied - #593 has the required update to the documentation.  When running `deploy all` the crds are installed.  

Are you able to pull the images directly to the host that you are running kind on?  And are you able to pull them to the kind node container?

**Comment by tobernguyen** [comment]
> @tobernguyen The initial error is because there is a new `crds` phase that also needs to be applied - #593 has the required update to the documentation. When running `deploy all` the crds are installed.
> 
> Are you able to pull the images directly to the host that you are running kind on? And are you able to pull them to the kind node container?

That makes sense. You are right about the missing `crds` phase. I'm going to add that to our minimum configuration. I tried to pull `quay.io/dexidp/dex:v2.27.0` and `docker.io/flanksource/platform-operator:0.5.1` locally and they don't exist. I believe this PR is trying to fix this problem by downgrading dex to 2.26.0: https://github.com/flanksource/karina/pull/652.
